insolar/node/node_db.go

```diff
@@ -0,0 +1,126 @@
+//
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package node
+
+import (
+	"github.com/dgraph-io/badger"
+	"github.com/insolar/insolar/insolar"
+	"github.com/insolar/insolar/insolar/store"
+)
+
+// StorageDB is a badger-based impl of a node storage
```

Comment:
Use `.`(dot) in the end of comments (Paragraph 7.1 of Code Style). Here and below.